src/get-started/install/_ios-setup.md

```diff
@@ -86,7 +86,35 @@ and an Apple Developer account. If your app is using Flutter plugins,
 you will also need the third-party CocoaPods dependency manager.
 
 <ol markdown="1">
+<li markdown="1">
+<a name="connect"></a>
+To set up physical device deployment in Xcode, 
+connect your device to the USB port on your 
+computer. 
+</li>
+<li markdown="1">
+<a name="wireless"></a>
+[Optional] To leverage wireless debugging, ensure that 
+your device is on the same network as your computer 
+and that the device has a set passcode.  
+
+While the device is attached, open **Xcode > Window > Devices and Simulators**. 
+Select your phone, and check **Connect via Network.**
 
+<img src="/assets/images/docs/setup/connect-wireless.png" alt="Check the box that says 'connect via network' from the devices and simulators page">
+
+Once the network icon appears next to the device name, 
+you can unplug your device from USB. 
+
+Sometimes it takes longer to find network devices. 
+If you don't see your device listed when using `flutter run`, 
+try extending the timeout: `flutter run --device-timeout 10`.
+
+For additional help troubleshooting, 
+check out [Apple's Developer Forums][]. For setting up 
```

Comment:
Linking to the top level forums is probably not very helpful, though I do appreciate that it's implying you need to get it working through Xcode before it will work with Flutter.

Reply:
Do you have a different link you think would be better? Or just remove